release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,14 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 30 October 2023 - Fix - CloudGateway 2.12.2 - Resolved an issue that could occur when using multiple CloudGateway modules [ID_37686]
+
+When multiple CloudGateway modules were installed in a cluster, they were not able to sync with each other anymore. In that case, the dataminer.services identity of the DMS was not synced correctly and only one CloudGateway would be able to function correctly, causing stability issues in the dataminer.services connection and features. This has now been resolved.
```

Comment:
I feel the two times "correctly" so short to each other reads a bit weird. Not sure how to properly word it myself tho

Reply:
maybe 'properly' :)